src/controllers/csv-processor.ts

```diff
@@ -381,15 +434,17 @@ export const processCSVFromBlobStorage = async (
             dataset_id: dataset.id
         };
     }
-    return processCSVData(buff, page, size, dataset, importObj);
+    return processSQLiteData(buff, page, size, dataset, importObj);
 };
```

Comment:
The functions `processSQLiteData`, `getColumnPreview`, `getFileFromDataLake`, etc seem to be unrelated to CSVs now - we should probably look at splitting this stuff out into a SQLite/cube specific module in future.